routing/cross_mwm_connector_serialization.hpp

```diff
@@ -47,21 +54,49 @@ class CrossMwmConnectorSerializer final
     }
 
     template <class Sink>
-    void Serialize(serial::CodingParams const & codingParams, uint32_t bitsPerOsmId,
+    void WriteCrossMwmId(connector::OsmId const & id, uint8_t n, BitWriter<Sink> & w) const
+    {
+      w.WriteAtMost64Bits(id.Get(), n);
+    }
+
+    template <class Sink>
+    void WriteCrossMwmId(connector::TransitId const & id, uint8_t /* n */, BitWriter<Sink> & w) const
```

Comment:
This seems strange - why do you ignore ```n```?

Reply:
Please have a look at my comment below and at the last commit about Header::m_bitsPerCrossMwmId.